src/js/utils/Select.jsx

```diff
@@ -221,7 +222,7 @@ class Select extends Component {
     const SingleValue = props => (
       <components.SingleValue {...props}>
         {this.props.valueRenderer ? (
-          this.props.valueRenderer(props.data)
+          this.props.valueRenderer({ ...props.data, showSelectedOptionColor })
```

Comment:
What hides under the `props.data`?
I'm wondering why you have to do it this way if e.g. `showLabelTooltip` is passed in the same way (using attributes) and it isn't provided here like this.

Reply:
props.data is a selected product, for example:
```
{
    "value": "ff8081816e1e6731016e1e6782610003",
    "id": "ff8081816e1e6731016e1e6782610003",
    "name": "Advil 200mg",
    "productCode": "00001",
    "handlingIcons": [],
    "lotAndExpiryControl": null,
    "displayName": null,
    "color": null,
    "exactMatch": true
}
```
I need this property in `SelectedValue` (in `ProductSelect` file), `showLabelTooltip` is not being used here. It is used within the `Select` component, but I need `showSelectedOptionColor` one level deeper.